arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/CallSection.java

```diff
@@ -385,35 +406,42 @@ public void resolveAtContextReEntry(Hub hub, CallFrame frame) {
       }
 
       case CALL_SMC_UNDEFINED -> {
+
+        // CALL_SMC_SUCCESS_XXX case
         if (successBit) {
           scenarioFragment.setScenario(CALL_SMC_SUCCESS_WONT_REVERT);
           return;
         }
 
+        // CALL_SMC_FAILURE_XXX case
         scenarioFragment.setScenario(CALL_SMC_FAILURE_WONT_REVERT);
 
         if (selfCallWithNonzeroValueTransfer) {
           childContextExitCallerSnapshot.decrementBalanceBy(value);
           reEntryCalleeSnapshot.decrementBalanceBy(value);
         }
 
+        int childId = hub.currentFrame().childFramesId().getLast();
+        CallFrame childFrame = hub.callStack().getById(childId);
+        int childContextRevertStamp = childFrame.revertStamp();
```

Comment:
**Note.** This revertStamp is that of a "self-induced" revert. Not the actual revert stamp of the child context. The child call / deployment may be successful (in which case we don't enter here) and be reverted later. Here we only care about children that are failures.